executor/conn.h

```diff
@@ -96,18 +96,23 @@ class Connection
 	static int Connect(const char* addr, const char* ports)
 	{
 		int port = atoi(ports);
+		bool localhost = !strcmp(addr, "localhost");
 		if (!strcmp(addr, "stdin"))
 			return STDIN_FILENO;
 		if (port == 0)
 			failmsg("failed to parse manager port", "port=%s", ports);
 		sockaddr_in saddr4 = {};
 		saddr4.sin_family = AF_INET;
 		saddr4.sin_port = htons(port);
+		if (localhost)
+			addr = "127.0.0.1";
 		if (inet_pton(AF_INET, addr, &saddr4.sin_addr))
```

Comment:
This call of inet_pton for localhost will be always successful, because it doesn't check whether ipv4 is supported, it is just convert a string ipv4 address to the binary format. Did I miss something?